### DISCUSSION TITLE
A question: how to set  line data structure for vector interpolation?

### DISCUSSION THREAD

**Discussion by 369386973** [nortikin/sverchok#4829] [Q&A] [answered]
![20221220115738](https://user-images.githubusercontent.com/101614185/208582253-e88f14d8-1894-43e9-a0d7-17f53ea04d63.jpg)
![20221220121359](https://user-images.githubusercontent.com/101614185/208582264-9ad934c1-2d91-4ffb-aa2f-3571a0593b59.jpg)

I want to achieve the effect in the second picture: the lines and its mirrors can be interpolated, but I don't know how to set the data structure correctly.

**Comment by Durman** [comment, marked as answer]
[untitled.blend.zip](https://github.com/nortikin/sverchok/files/10284271/untitled.blend.zip)

![image](https://user-images.githubusercontent.com/28003269/209069986-f2c3ec19-c062-42cc-b06c-c8ead894a1a3.png)

**Reply by 369386973** [reply]
An easy way to understand. Thx！
